neuralmonkey/decoding_function.py

```diff
@@ -47,6 +48,12 @@ def attention_decoder(decoder_inputs, initial_state, attention_objects,
             outputs.append(output)
             states.append(state)
 
+        if summary_collections:
+            for i, a in enumerate(attention_objects):
+                alignments = tf.expand_dims(tf.transpose(tf.pack(a.attentions_in_time), perm=[1, 2, 0]), -1)
```

Comment:
just FYI, the line length used by me is 80. I will add it to the documentation for developers